### PR TITLE
Fix js code sample

### PR DESCRIPTION
**Pull request by hoffmabc**

### CODE REVIEW

**Comment by diasdavid** [comment]
thank you @hoffmabc :)

**Comment by hoffmabc** [comment]
Always making big impacts

File: README.md [hunk 1]
```diff
@@ -120,9 +120,9 @@ Example:
 
 ```js
 var fsBlobStore = require('fs-blob-store')  // an in-memory blob store
```
Comment (dignifiedquire) [inline README.md]: actually this is not correct anymore, it needs to be `fs-pull-blob-store` and the comment is wrong as this is not an in-memory blob store, but an fs blob store

Comment (dignifiedquire) [inline README.md]: cc @diasdavid

Comment (diasdavid) [inline README.md]: true, should had tested it. Fixed it now https://github.com/ipfs/js-ipfs-repo/commit/c2c76d7df55381e89f3a3558b9f98962e3a1dd46

**Pull request merged**